src/univariate/discrete/discretenormal.jl

```diff
@@ -0,0 +1,250 @@
+"""
+Represents the (truncated) discrete normal distribution.
+
+For an integer `x` in the support, the probability is proportional to
+
+    P(x) = ℯ^(-(x-μ)^2/2σ^2)
```

Comment:
```suggestion
    `P(x; µ, σ) = ℯ^(-(x-μ)^2/2σ^2)`
```

Reply:
The backticks there in the suggestion are unnecessary, since indenting by 4 spaces creates a code block in markdown.

Reply:
My bad thanks, I edited the suggestion